tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "declaration": true,
     "declarationDir": "lib/types",
     "noEmit": true,
-    "lib": ["es2017", "ESNext.AsyncIterable", "dom", "webworker"],
+    "lib": ["DOM", "DOM.Iterable", "ESNext.AsyncIterable"],
```

Comment:
TIL, use `dom.iterable` to get iterable methods like `keys()`, `values()`, and `entries()` on the `Headers` class.